rpcs3/Emu/Cell/lv2/sys_spu.cpp

```diff
@@ -306,7 +306,12 @@ error_code sys_spu_thread_group_create(vm::ptr<u32> id, u32 num, s32 prio, vm::p
 		sys_spu.todo("Unsupported SPU Thread Group type (0x%x)", attr->type);
 	}
 
-	*id = idm::make<lv2_spu_group>(std::string(attr->name.get_ptr(), attr->nsize - 1), num, prio, attr->type, attr->ct);
+	if (attr->nsize > 0x80)
```

Comment:
why not check for this in the parameter check block above? it already checks for the oter stuff including attr->type so I feel this check belongs there as well